jabgui/src/main/java/org/jabref/gui/newentry/NewEntryView.java

```diff
@@ -143,7 +145,21 @@ public NewEntryView(NewEntryDialogTab initialApproach, GuiPreferences preference
     private void finalizeTabs() {
         NewEntryDialogTab approach = initialApproach;
         if (approach == null) {
-            approach = preferences.getLatestApproach();
+            final String clipboardText = ClipBoardManager.getContents().trim();
+            if (!StringUtil.isBlank(clipboardText)) {
+                BibEntry tempEntry = new BibEntry().withField(StandardField.DOI, clipboardText);
+                IdentifierParser parser = new IdentifierParser(tempEntry);
+        
+                if (parser.parse(StandardField.DOI).isPresent()) {
```

Comment:
Can you please add a clarifying comment here that we put every kind of identifier from the clipboard in the temp DOI field because it also works for isbn or arxiv